### PR TITLE
fix (v8): update more .npmignore files to resolve node16 regression with npm publishing

### LINKED ISSUE #27732: [Bug]: ESM and AMD folders are missing in the 8.108.3 update
### Library

React / v8 (@fluentui/react)

### System Info

```shell
System:
    OS: macOS 13.3.1
    CPU: (10) arm64 Apple M1 Pro
    Memory: 142.19 MB / 16.00 GB
    Shell: 5.9 - /bin/zsh
  Browsers:
    Chrome: 112.0.5615.137
    Chrome Canary: 115.0.5743.0
    Edge: 112.0.1722.64
    Firefox: 112.0.2
    Safari: 16.4
```

### Are you reporting Accessibility issue?

no

### Reproduction

https://aka.ms/fluentpen

## Actual Behavior
The lib (esm) and lib-amd folders are missing in the v8.108.3, this is causing a dependency bug for all version 8 lib.
https://www.npmjs.com/package/@fluentui/react/v/8.108.3?activeTab=code

## Expected Behavior
These folders should present like:
https://www.npmjs.com/package/@fluentui/react/v/8.108.2?activeTab=code

### Logs

_No response_

### Requested priority

High

### Products/sites affected

fluentui/react^8

### Are you willing to submit a PR to fix?

no

### Validations

- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] The provided reproduction is a minimal reproducible example of the bug.

### PR DESCRIPTION
**Pull request by TristanWatanabe**
## Changes:
- some v8 packages were missed in the updates from last night so this PR updates the `.npmignore` file of packages missed.

## Related Issue(s)

Follows #27736
Fixes #27732

### CODE REVIEW

File: packages/react/.npmignore [hunk 1]
```diff
@@ -33,3 +33,4 @@ visualtests
 !lib
 !lib-commonjs
 !lib-amd
+!dist
```
Comment (TristanWatanabe) [inline packages/react/.npmignore:36]: `@fluentui/react` makes use of the `dist/` folder as well which was missed in last night's publishing so adding here

File: change/@fluentui-azure-themes-8c43c313-f51c-4342-9d83-4e1b5780d129.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (fabricteam) [inline change/@fluentui-azure-themes-8c43c313-f51c-4342-9d83-4e1b5780d129.json:1]: ## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the 2 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=27745&commitId=05bddcb1e3b8dca8c33b2c13c1323fb4f9fd0ec2&env=prod)

✅ There was 2 screenshots added, 0 screenshots removed, 1045 screenshots unchanged, 0 screenshots with different dimensions and 0 screenshots with visible difference.


  <details><summary><b>unknown</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Pivot - Overflow.Tabs - RTL.Narrow - Last tab selected.chromium.png|0| Added|
|Pivot - Overflow.Tabs - RTL.Narrow - Overflow menu.chromium.png|0| Added|

</details>


<div id="vrtCommentfluentuiv8"/>

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/2e4a9ee0c93e252ac19771d934b5f9e233faddd7'>2e4a9ee0c93e252ac19771d934b5f9e233faddd7</a></sub>
<div id="bundleSizeComment9423"/>

**Review by JustSlone — APPROVED** [review]

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>BaseButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27745/perf-test/BaseButton-mount_base.html">913</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27745/perf-test/BaseButton-mount.html">906</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Breadcrumb</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27745/perf-test/Breadcrumb-mount_base.html">2540</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27745/perf-test/Breadcrumb-mount.html">2554</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Checkbox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27745/perf-test/Checkbox-mount_base.html">2463</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27745/perf-test/Checkbox-mount.html">2436</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>CheckboxBase</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27745/perf-test/CheckboxBase-mount_base.html">2119</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27745/perf-test/CheckboxBase-mount.html">2151</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ChoiceGroup</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27745/perf-test/ChoiceGroup-mount_base.html">4214</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27745/perf-test/ChoiceGroup-mount.html">4221</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ComboBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27745/perf-test/ComboBox-mount_base.html">989</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27745/perf-test/ComboBox-mount.html">984</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>CommandBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27745/perf-test/CommandBar-mount_base.html">9070</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27745/perf-test/CommandBar-mount.html">9116</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>ContextualMenu</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27745/perf-test/ContextualMenu-mount_base.html">16647</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27745/perf-test/ContextualMenu-mount.html">16947</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DefaultButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27745/perf-test/DefaultButton-mount_base.html">1129</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27745/perf-test/DefaultButton-mount.html">1122</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRow</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27745/perf-test/DetailsRow-mount_base.html">3329</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27745/perf-test/DetailsRow-mount.html">3277</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowFast</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27745/perf-test/DetailsRowFast-mount_base.html">3315</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27745/perf-test/DetailsRowFast-mount.html">3283</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowNoStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27745/perf-test/DetailsRowNoStyles-mount_base.html">3037</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27745/perf-test/DetailsRowNoStyles-mount.html">3079</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Dialog</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27745/perf-test/Dialog-mount_base.html">3906</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27745/perf-test/Dialog-mount.html">3899</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DocumentCardTitle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27745/perf-test/DocumentCardTitle-mount_base.html">360</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27745/perf-test/DocumentCardTitle-mount.html">353</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Dropdown</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27745/perf-test/Dropdown-mount_base.html">2912</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27745/perf-test/Dropdown-mount.html">2920</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusTrapZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27745/perf-test/FocusTrapZone-mount_base.html">1710</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27745/perf-test/FocusTrapZone-mount.html">1677</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27745/perf-test/FocusZone-mount_base.html">1657</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27745/perf-test/FocusZone-mount.html">1639</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27745/perf-test/GroupedList-mount_base.html">55938</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27745/perf-test/GroupedList-mount.html">64726</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27745/perf-test/GroupedList-virtual-rerender_base.html">24485</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27745/perf-test/GroupedList-virtual-rerender.html">27350</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27745/perf-test/GroupedList-virtual-rerender-with-unmount_base.html">83446</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27745/perf-test/GroupedList-virtual-rerender-with-unmount.html">83956</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27745/perf-test/GroupedListV2-mount_base.html">347</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27745/perf-test/GroupedListV2-mount.html">366</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27745/perf-test/GroupedListV2-virtual-rerender_base.html">330</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27745/perf-test/GroupedListV2-virtual-rerender.html">340</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27745/perf-test/GroupedListV2-virtual-rerender-with-unmount_base.html">353</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27745/perf-test/GroupedListV2-virtual-rerender-with-unmount.html">360</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>IconButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27745/perf-test/IconButton-mount_base.html">1621</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27745/perf-test/IconButton-mount.html">1636</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Label</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27745/perf-test/Label-mount_base.html">517</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27745/perf-test/Label-mount.html">521</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Layer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27745/perf-test/Layer-mount_base.html">4116</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27745/perf-test/Layer-mount.html">4055</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Link</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27745/perf-test/Link-mount_base.html">590</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27745/perf-test/Link-mount.html">591</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MenuButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27745/perf-test/MenuButton-mount_base.html">1380</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27745/perf-test/MenuButton-mount.html">1405</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MessageBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27745/perf-test/MessageBar-mount_base.html">31721</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27745/perf-test/MessageBar-mount.html">31865</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Nav</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27745/perf-test/Nav-mount_base.html">2877</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27745/perf-test/Nav-mount.html">2912</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>OverflowSet</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27745/perf-test/OverflowSet-mount_base.html">1140</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27745/perf-test/OverflowSet-mount.html">1135</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Panel</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27745/perf-test/Panel-mount_base.html">2548</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27745/perf-test/Panel-mount.html">2573</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27745/perf-test/Persona-mount_base.html">1085</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27745/perf-test/Persona-mount.html">1134</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Pivot</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27745/perf-test/Pivot-mount_base.html">1358</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27745/perf-test/Pivot-mount.html">1341</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>PrimaryButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27745/perf-test/PrimaryButton-mount_base.html">1244</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27745/perf-test/PrimaryButton-mount.html">1235</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Rating</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27745/perf-test/Rating-mount_base.html">6584</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27745/perf-test/Rating-mount.html">6580</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SearchBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27745/perf-test/SearchBox-mount_base.html">1329</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27745/perf-test/SearchBox-mount.html">1289</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Shimmer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27745/perf-test/Shimmer-mount_base.html">2643</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27745/perf-test/Shimmer-mount.html">2643</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Slider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27745/perf-test/Slider-mount_base.html">1973</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27745/perf-test/Slider-mount.html">2010</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27745/perf-test/SpinButton-mount_base.html">4243</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27745/perf-test/SpinButton-mount.html">4276</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Spinner</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27745/perf-test/Spinner-mount_base.html">587</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27745/perf-test/Spinner-mount.html">597</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SplitButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27745/perf-test/SplitButton-mount_base.html">2795</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27745/perf-test/SplitButton-mount.html">2752</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Stack</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27745/perf-test/Stack-mount_base.html">597</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27745/perf-test/Stack-mount.html">604</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithIntrinsicChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27745/perf-test/StackWithIntrinsicChildren-mount_base.html">1277</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27745/perf-test/StackWithIntrinsicChildren-mount.html">1286</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithTextChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27745/perf-test/StackWithTextChildren-mount_base.html">3707</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27745/perf-test/StackWithTextChildren-mount.html">3678</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SwatchColorPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27745/perf-test/SwatchColorPicker-mount_base.html">9229</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27745/perf-test/SwatchColorPicker-mount.html">9241</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TagPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27745/perf-test/TagPicker-mount_base.html">2254</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27745/perf-test/TagPicker-mount.html">2285</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Text</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27745/perf-test/Text-mount_base.html">578</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27745/perf-test/Text-mount.html">556</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TextField</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27745/perf-test/TextField-mount_base.html">1370</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27745/perf-test/TextField-mount.html">1403</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27745/perf-test/ThemeProvider-mount_base.html">1280</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27745/perf-test/ThemeProvider-mount.html">1296</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27745/perf-test/ThemeProvider-virtual-rerender_base.html">888</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27745/perf-test/ThemeProvider-virtual-rerender.html">883</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27745/perf-test/ThemeProvider-virtual-rerender-with-unmount_base.html">1974</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27745/perf-test/ThemeProvider-virtual-rerender-with-unmount.html">1963</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Toggle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27745/perf-test/Toggle-mount_base.html">905</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27745/perf-test/Toggle-mount.html">909</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>buttonNative</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27745/perf-test/buttonNative-mount_base.html">319</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27745/perf-test/buttonNative-mount.html">316</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9423"/>

**Pull request merged**